tap_typeform/streams.py

```diff
@@ -272,9 +281,19 @@ def get_bookmark_value(state, stream_name, key, default=None):
 
 def sync_landings(atx, form_id):
     response = get_landings(atx, form_id)
+    data = response.get('items', [])
+
+    page_count = response.get('page_count', 1)
+    token = data[-1].get('token')
+
+    while page_count > 1:
+        response = get_landings(atx, form_id, token, next_page=True)
+        page_count = response.get('page_count', 1)
+        data.extend(response.get('items'), [])
```

Comment:
I think this is the wrong approach. We could hit a memory limit by holding on to every page with this `extend`.

Is there an advantage to this over writing a page of records at a time?